browser/extensions/brave_component_loader.cc

```diff
@@ -47,7 +50,12 @@ void BraveComponentLoader::OnComponentReady(std::string extension_id,
     bool allow_file_access,
     const base::FilePath& install_dir,
     const std::string& manifest) {
-  Add(manifest, install_dir);
+  if (extension_id == ethereum_remote_client_extension_id) {
```

Comment:
This is done so that it downloads and installs the component but doesn't actually load it to use the background page until the user first accesses brave://wallet